src/commands/createStaticWebApp/PresetBuildListStep.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { IPresetBuild } from '../../presetBuilds/IPresetBuild';
+import { presetBuilds } from '../../presetBuilds/presetBuilds';
+import { localize } from '../../utils/localize';
+import { openUrl } from '../../utils/openUrl';
+import { ApiLocationStep } from './ApiLocationStep';
+import { AppArtifactLocationStep } from './AppArtifactLocationStep';
+import { AppLocationStep } from './AppLocationStep';
+import { IStaticWebAppWizardContext } from './IStaticWebAppWizardContext';
+
+export class PresetBuildListStep extends AzureWizardPromptStep<IStaticWebAppWizardContext> {
+
+    public async prompt(context: IStaticWebAppWizardContext): Promise<void> {
+        const placeHolder: string = localize('choosePreset', 'Choose preset build to configure default project structure');
+        const picks: IAzureQuickPickItem<IPresetBuild | undefined>[] = presetBuilds.map((pb) => { return { label: pb.displayName, data: pb }; });
+        picks.push({ label: 'Custom', data: undefined });
+        const learnMore: IAzureQuickPickItem = { label: localize('learnMore', '$(link-external) Learn more...'), description: '', data: undefined };
+        picks.push(learnMore);
+        let pick: IAzureQuickPickItem<IPresetBuild | undefined>;
+
+        do {
+            pick = await ext.ui.showQuickPick(picks, { placeHolder, suppressPersistence: true });
+            if (pick === learnMore) {
+                await openUrl('https://docs.microsoft.com/en-us/azure/static-web-apps/front-end-frameworks');
+            }
+        } while (pick === learnMore);
+
+        if (pick.data) {
+            context.appLocation = pick.data.appLocation;
+            context.apiLocation = pick.data.apiLocation;
+            context.appArtifactLocation = pick.data.appArtifactLocation;
+        }
+
+    }
+
+    public shouldPrompt(context: IStaticWebAppWizardContext): boolean {
+        return !context.appLocation;
+    }
+
+    public async getSubWizard(context: IStaticWebAppWizardContext): Promise<IWizardOptions<IStaticWebAppWizardContext> | undefined> {
+        if (!context.appLocation) {
+            return { promptSteps: [new AppLocationStep(), new ApiLocationStep(), new AppArtifactLocationStep()], executeSteps: [] };
```

Comment:
Is it required to explicitly set `executeSteps: []`? I thought that was undefined by default